modules/eco-node-maintenance-operator-installation-cli.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-maintenance-operator-using-cli_{context}"]
+= Installing the Node Maintenance Operator by using the CLI
+You can use the OpenShift CLI (`oc`) to install the Node Maintenance Operator.
+
+You can install the Node Maintenance Operator in your own namespace or in the `openshift-operators` namespace. 
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+To install the Operator in your own namespace, follow the steps in the procedure. 
+
+To install the Operator in the `openshift-operators` namespace, skip to step 3 of the procedure because the steps to create to create a new `Namespace` custom resource (CR) and a `OperatorGroup` CR are not required. 
```

Comment:
Yeah, I had used "as" but later when I check the ISG Word Usage for [as](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#as), I changed it to "because"